examples/scott.m31

```diff
@@ -25,28 +25,19 @@ do (λ f, refl (Y f)) : Π f : D → D, Y f ≡ f (Y f)
    happen at [D], but the application at [D']. *)
 
 constant D' : Type
-constant h : D ≡ D'
-let scott' =
-  now hints = add_hint h in
-  scott : (D' → D') ≡ D'
-
-now hints = add_hint scott'
+constant D'_eq_D : D' ≡ D
 
 (* The λ-abstraction happens at type [D] *)
-let δ = 
-  (λ x, (x : D → D) x)
-  : D → D
+let δ = (λ x : D, (x : D → D) x)
 
 (* But we make sure that the application δ δ happens at [D']. *)
 let Ω =
-    now hints = add_hint h in
-    let δ' = δ : D' → D' in
-    now betas = add_betas [h,scott'] in
-      δ' (δ' : D')
+  now hints = add_hints [D'_eq_D, symmetry D'_eq_D] in
+  (δ : D' → D') (δ : D)
+
+do whnf Ω
 
 (* Notice that [Ω] does not diverge even though a beta-reduction happens. *)
 do refl Ω : (λ x : D', x) Ω ≡ Ω
 
-do external "exit" ()
-
-do refl (δ δ) : (λ x : D, x) (δ δ) ≡ (δ δ)
+(* do refl (δ δ) : (λ x : D, x) (δ δ) ≡ (δ δ) *)
```

Comment:
Use `let _ = fun _ => refl ...` (anonymous hunk) to detect changes in parsing/typing when running the example.